### PR TITLE
definitions for angular-gettext v2.1.0

### PR DESCRIPTION
**Pull request by AkosLukacs**
https://angular-gettext.rocketeer.be/

### CODE REVIEW

**Comment by vvakame** [comment]
*angular-gettext/angular-gettext.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/angular-gettext
  * http://bower.io/search/?q=angular-gettext
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (angular-gettext/angular-gettext-tests.ts or others)
* [ ] pass the Travic-CI test?

File: angular-gettext/angular-gettext.d.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿// Type definitions for angular-gettext v2.1.0
+// Project: https://angular-gettext.rocketeer.be/
+// Definitions by: Ákos Lukács <https://github.com/AkosLukacs>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../angularjs/angular.d.ts" />
+
+declare module angular_gettext {
```
Comment (vvakame) [inline angular-gettext/angular-gettext.d.ts]: I think `angular.gettext` is better name.

**Comment by dmandalidis** [comment]
Hi all, is this going to be merged soon? is it considered ready?

**Comment by AkosLukacs** [comment]
It is ready. At least I think so...

**Comment by vvakame** [comment]
@dmandalidis @AkosLukacs oops, it merged. thanks mate!

**Pull request merged**